docs/user/hbase/install.rst

```diff
@@ -53,6 +53,133 @@ distribution. If you have built from source, the distribution is created in the
 
 More information about developing with GeoMesa may be found in the :doc:`/developer/index`.
 
+.. _registering_coprocessors:
+
+Register the Coprocessors
+-------------------------
+
+GeoMesa utilizes server side processing to accelerate some queries. Currently the only processing done server side is
+density (heatmap) calculations. In order to utilize this feature the GeoMesa coprocessor must be registered on all GeoMesa tables
```

Comment:
Installing the jar allows the remote GeoMesa filters to be used.  Without the hbase-site.xml configuration, the coprocessors may not be used.